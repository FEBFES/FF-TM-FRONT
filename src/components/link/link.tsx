import React from "react"

const Link = () => {
	return <a href=""></a>
}

export default Link
