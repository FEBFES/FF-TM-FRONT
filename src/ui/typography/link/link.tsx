import React, { DetailedHTMLProps } from 'react';

interface LinkProps
  extends DetailedHTMLProps<
    React.AnchorHTMLAttributes<HTMLAnchorElement>,
    HTMLAnchorElement
  > {
  children: React.ReactNode;
}

export const Link: React.FC<LinkProps> = ({ children, ...props }) => {
  return (
    <a {...props} className={props.className}>
      {children}
    </a>
  );
};
