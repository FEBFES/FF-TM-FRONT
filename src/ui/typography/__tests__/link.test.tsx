import React from 'react';
import { Link } from '../link/link';
import renderer from 'react-test-renderer';

it('SNAPSHOTS: UI - Link component - render', () => {
  const elem = renderer.create(<Link>linka</Link>).toJSON();
  expect(elem).toMatchSnapshot();
});
