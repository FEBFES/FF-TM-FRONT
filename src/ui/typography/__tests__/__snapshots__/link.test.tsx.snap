// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SNAPSHOTS: UI - Link component - render 1`] = `
<a>
  linka
</a>
`;
